modules/installation-launching-installer.adoc

```diff
@@ -68,6 +68,17 @@ process uses to the installation program.
 .. If you do not have an AWS profile stored on your computer, enter the AWS
 access key ID and secret access key for the user that you configured to run the
 installation program.
++
+[IMPORTANT]
+====
+You must not use a temporary session token that you generated while using a
```

Comment:
Would it be better to put this in the prerequisites where we discuss obtaining the access token?

Reply:
It's not this access token. :)
I wish that they hadn't made the installer smart enough to generate your credentials for you - creating the credentials was briefly a separate step in its own module, and that's where it would make most sense. Supplying your AWS creds is not technically a prereq at all. I think your suggestion's good, though, so I'm moving this information to the assembly prereqs.